scripts/local-clusters/resolves/Corefile

```diff
@@ -1,10 +1,15 @@
 .:53 {
   errors
   template IN A {$domain} {
-    match "\.{$domain}\.$"
+    match "(grafana|harbor|opensearch|ops)\.{$domain}\.$"
```

Comment:
Observation: Would be problematic if those subdomains are configured to something else, but maybe that's fine for a dev environment?

Maybe, to be more exact (e.g. avoid matching `someops.$domain`):

```suggestion
    match "(^grafana|^harbor|^opensearch|\.ops)\.{$domain}\.$"
```